co-simulation/fed/mosaic-carma-cloud/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+        <groupId>org.eclipse.mosaic</groupId>
+        <artifactId>mosaic-parent</artifactId>
+        <version>22.1-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>mosaic-carma-cloud</artifactId>
+  <name>CARMA Cloud Ambassador</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.eclipse.mosaic</groupId>
+      <artifactId>mosaic-rti-api</artifactId>
+      <version>${mosaic.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.mosaic</groupId>
+      <artifactId>mosaic-objects</artifactId>
+      <version>${mosaic.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.mosaic</groupId>
+      <artifactId>mosaic-objects</artifactId>
+      <version>${mosaic.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>gov.dot.fhwa.saxton</groupId>
+        <artifactId>mosaic-carma-utils</artifactId>
+        <version>${mosaic.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.mosaic</groupId>
+      <artifactId>mosaic-interactions</artifactId>
+      <version>${mosaic.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.mosaic</groupId>
+      <artifactId>mosaic-utils</artifactId>
+      <version>${mosaic.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.mosaic</groupId>
+      <artifactId>mosaic-geomath</artifactId>
+      <version>${mosaic.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
+      <scope>test</scope>
+    </dependency>
+      <dependency>
+          <groupId>org.eclipse.mosaic</groupId>
+          <artifactId>mosaic-application</artifactId>
+          <version>22.1-SNAPSHOT</version>
+          <scope>compile</scope>
+      </dependency>
+      <dependency>
+          <groupId>jakarta.xml.bind</groupId>
+          <artifactId>jakarta.xml.bind-api</artifactId>
+          <version>2.3.2</version>
+      </dependency>
+      <dependency>
+          <groupId>org.glassfish.jaxb</groupId>
```

Comment:
What is this dependency necessary for?

Reply:
It appears Cheng fixed this.

Reply:
What has been fixed. I was asking what this dependency is used for. Is it being used somewhere